moveit_commander/test/python_moveit_commander.py

```diff
@@ -94,6 +94,12 @@ def test_validation(self):
     def test_planning_scene_interface(self):
         planning_scene = PlanningSceneInterface()
 
+    def test_get_jacobian_matrix(self):
+        current = self.group.get_current_joint_values()
+        matrix = np.array(self.group.get_jacobian_matrix(current))
+        self.assertEqual(matrix.shape[0], 6)
+        self.assertEqual(matrix.shape[1], 6)
+
```

Comment:
Thank you for adding a simple test for it.
Why did you duplicate the same test code 4 times though?

Testing the dimensions of the Jacobian makes sense, but I would also suggest to test the (known) value of a specific arm configuration. Would that be too much work/ too unstable to add?

Reply:
> Why did you duplicate the same test code 4 times though?

Yes, I felt strange but the other tests have same 4 entry in each file, so I just mimicked them. 
move_group and moveit_commander have same tests and I guess `_ns` tests are for namespace enabled. But if I don't need 4 copies, I want to add only one (I'm not sure which file  is appropriate to add).

> I would also suggest to test the (known) value of a specific arm configuration.

Agreed, it is not difficult to test the known values. Thanks.

Reply:
@mlautman, @rhaschke, @bmagyar I'm not sure who worked on these tests, I guess one of you.
Could anyone of you comment on the current copy&paste situation?

Reply:
Indeed this code duplication is horrible. For sure the duplication between `moveit_commander/test` and `moveit_ros/planning_interface/test` is superfluous. Also, the namespace variant could be better handled via an argument to the correponding `.test` file. Looks like @willcbaker introduced these duplicates in #835. @willcbaker, could you please clean this up?

Reply:
yup, definitely cringe-worthy

Reply:
yup, definitely cringe-worthy

Reply:
I removed duplicated my tests. Only test in `moveit_ros/planning_interface/test/python_move_group.py` remains.

@v4hn Thanks, I revised to check the jacobian matrix  by known values at initial pose.